### DISCUSSION TITLE
ARCropolis has panicked

### DISCUSSION THREAD

**Discussion by carmr** [Raytwo/ARCropolis#518] [Issues]
I got this error message before and after uninstalling and reinstalling everything to do with ARCropolis and Skyline. Any help or advice would be appreciated.
![2023040501310900-1AB131B6E6571375B79964211BB3F5AE](https://user-images.githubusercontent.com/114680131/230004058-a2a64bf5-b72b-4c83-840c-9b5d5bc409cc.jpg)

**Comment by Kybo27** [comment]
I have exactly the same problem
